src/test/java/uk/gov/hmcts/reform/em/orchestrator/automatedbundling/AutomatedCaseUpdaterTest.java

```diff
@@ -123,6 +193,26 @@ public void createCaseBundlesPropertyWhenItDoesntExist() throws IOException {
         assertEquals(1, bundles.get().size());
     }
 
+    @Ignore
```

Comment:
For now, pls remove Ignored test cases. As we have to give unnecessary justification to release team why this test been ignored.